Resources/Prototypes/_Mono/Shipyard/BlackMarket/mercury.yml

```
@@ -0,0 +1,32 @@
+  stations:
+    Mercury:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Mercury {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Pirate: [ 0, 0 ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)